xdsl/dialects/arith.py

```diff
@@ -271,7 +273,7 @@ def get_canonicalization_patterns(cls) -> tuple[RewritePattern, ...]:
 
 
 @irdl_op_definition
-class Addi(SignlessIntegerBinaryOp):
+class Addi(SignlessIntegerBinaryOperation):
```

Comment:
Any particular reason to change the name here?

Reply:
The undocumented recommendation is for ABCs to have the full Operation and concrete ops to have Op as a suffix